Divide_Conquer/315.Count-of-Smaller-Numbers-After-Self/315.Count-of-Smaller-Numbers-After-Self.cpp

```diff
@@ -33,4 +33,90 @@ class Solution {
         // 如果写归并排序的code会更快一些。这里就偷懒了，直接用sort函数。        
         sort(sortedNums.begin()+start,sortedNums.begin()+end+1);        
     }
+
+  std::vector<int> count_smaller_binary_indexed_tree(std::vector<int> &nums)
```

Comment:
Would you add a new file like 315.Count-of-Smaller-Numbers-After-Self_v2.cpp in this directory and put your code there? Thank you.